### PR TITLE
Forward port release notes for v0.18.2

### PR DESCRIPTION
**Pull request by jni**
'nuff said.

### CODE REVIEW

File: doc/release/release_0.18.rst [hunk 1]
```diff
@@ -1,3 +1,59 @@
...
+- add numpy version specification on aarch for cpython 3.8 (#5374, #5375)
+
+7 authors added to this release [alphabetical by first name or login]
+---------------------------------------------------------------------
+- François Boulogne
+- Janakarajan Natarajan
+- Juan Nunez-Iglesias
+- John Lee
+- Mark Harfouche
+- MeeseeksMachine
```
Comment (mkcor) [inline doc/release/release_0.18.rst:40]: Should this bot be included as an author?

File: doc/release/release_0.18.rst [hunk 2]
```diff
@@ -1,3 +1,59 @@
...
+===================
+
+We're happy to announce the release of scikit-image v0.18.2!
+
+scikit-image is an image processing toolbox for SciPy that includes algorithms
+for segmentation, geometric transformations, color space manipulation,
+analysis, filtering, morphology, feature detection, and more.
+
+This release mostly serves to add wheels for the aarch64 architecture, as well
+as fixing a couple of minor bugs.
```
Comment (mkcor) [inline doc/release/release_0.18.rst:11]: "(...) serves to add (...) as well as (to) fix (...)" although it doesn't really *serve* to fix, rather it *happens* to fix minor bugs as well... :thinking: 
Maybe: "This release mostly serves to add wheels for the aarch64 architecture; it also
fixes a couple of minor bugs."

File: doc/release/release_0.18.rst (hunk 1 above)
Comment (jni) [reply doc/release/release_0.18.rst:40]: I like to give it credit. :smiley: We should keep the robots happy to postpone the robot uprising! :joy:

File: doc/release/release_0.18.rst (hunk 2 above)
Comment (jni) [reply doc/release/release_0.18.rst:11]: Can you make that a suggestion so I can click and commit? :smile:

File: doc/release/release_0.18.rst (hunk 1 above)
Comment (mkcor) [reply doc/release/release_0.18.rst:40]: Ok, ok!!

File: doc/release/release_0.18.rst (hunk 2 above)
Comment (mkcor) [inline doc/release/release_0.18.rst:11]:
```suggestion
This release mostly serves to add wheels for the aarch64 architecture; it also
fixes a couple of minor bugs.
```

**Review by mkcor — APPROVED** [review]

File: doc/release/release_0.18.rst (hunk 2 above)
Comment (mkcor) [reply doc/release/release_0.18.rst:11]: Sure! Approving with the suggested change.

**Review by rfezzani — APPROVED** [review]

**Pull request merged**